### DISCUSSION TITLE
feat: add numberOfLines props to Snackbar

### DISCUSSION THREAD

**Discussion by whoami-shubham** [callstack/react-native-paper#2597] [Ideas]
Right now I couldn't find any props for trimming the snack bar text, I can trim the text before giving it to the snack bar however that wouldn't be very responsive. if the text is very long the snack bar looks like following
![image](https://user-images.githubusercontent.com/28999685/110437915-e306d100-80db-11eb-953b-01d2ba225791.png)

if I provide numberOfLines = 2 at [here](https://github.com/callstack/react-native-paper/blob/975cfa3be4956b629aa8255e226ff9ca068448eb/src/components/Snackbar.tsx#L205)  it looks like following 
![image](https://user-images.githubusercontent.com/28999685/110438381-527cc080-80dc-11eb-95a0-cf3d13a18a55.png)

let me know is it a good idea to work on it.

1. react-native : 0.63.3
2. react-native-paper: 4.2.0
3. react-native-vector-icons:6.6.0

**Comment by raajnadar** [comment]
Yes this looks ok to me.

**Reply by whoami-shubham** [reply]
should I send a PR for it ?

**Reply by raajnadar** [reply]
Yes, And let the internal team decide if that is ok
